docs/middleware/index.md

```diff
@@ -132,6 +135,44 @@ This action has these additional `context` properties:
 
 > The presence object being sent. Its shape depends on its [type]({{ site.baseurl }}{% link types/index.md %})
 
+### `'query'`
+
+A new query request is about to be submitted to the database
+
+This action has these additional `context` properties:
+
+`index` -- string
+
+> The name of the query's collection or [projection]({{ site.baseurl }}{% link projections.md %})
+
+`collection` -- string
+
+> The query's target collection
+
+`projection` -- string
+
+> The query's [projection]({{ site.baseurl }}{% link projections.md %})
+
+`fields` -- Object
+
+> The query's projection fields
```

Comment:
Do you have an example of this? I can't parse the difference between "projection" and "projection fields"